tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -76,6 +78,8 @@ def setUp(self):
                     project_id=PROJECT_ID,
                     num_workers=NUM_WORKERS,
                     zone=ZONE,
+                    subnetwork_uri=SUBNETWORK_URI,
```

Comment:
Any need to test network URI vs subnetwork URI?

Reply:
Usually you'd have only one or the other so I chose subnetworkUri